mongodb/activity/queryDocument/activity.go

```diff
@@ -0,0 +1,209 @@
+package queryDocument
+
+import (
+	ctx "context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	mongocon "github.com/project-flogo/datastore-contrib/mongodb/connector/connection"
+	// "github.com/TIBCOSoftware/flogo-lib/core/activity"
+	// "github.com/TIBCOSoftware/flogo-lib/core/data"
+	// "github.com/TIBCOSoftware/flogo-lib/logger"
+	//"github.com/TIBCOSoftware/flogo-lib/core/data"
+	//"github.com/TIBCOSoftware/flogo-lib/core/data"
+	// Need to remove this after testing with latest API
+	"github.com/project-flogo/core/activity"
+	"github.com/project-flogo/core/support/log"
+
+	//"github.com/project-flogo/core/data/metadata"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// var log = logger.GetLogger("activity-mongodb")
+
+const (
+	connectionProp = "mongoConnection"
+	ivOperation    = "operation"
+	outputProperty = "Output"
+	ivInput        = "input"
+	ovCount        = "count"
+)
+
+func init() {
+	fmt.Println("Entered the init")
+	err := activity.Register(&Activity{}, New)
+	if err != nil {
+		fmt.Println(err)
+	}
+}
+
+// New functioncommon
+func New(ctx1 activity.InitContext) (activity.Activity, error) {
```

Comment:
It's better if it's just `ctx` rather than  `ctx1` .

Reply:
yes, will change